site.yml

```diff
@@ -3,10 +3,32 @@
 
 - hosts: "{{ hosts_prefix }}-tsuru-db"
   sudo: yes
+  vars:
+    mongodb_conf_dbpath: "/var/lib/mongodb"
   roles:
     - bennojoy.redis
     - greendayonfire.mongodb
 
+# MongoDB: migrate old /data/db dir to new location
```

Comment:
I think the tasks already sufficiently describe what this is doing. However I think it'd be useful to have a comment of `FIXME: Remove when deployed to all environments` so that we can a) search for FIXME to find code that we no longer need, and b) know that we can remove these tasks at some point in the near future (and under what conditions).